bin/compile_model.sh

```diff
@@ -203,12 +211,18 @@ elif [ "$SITE" == "alvin" ] || [ "$SITE" == "elvis" ] ; then
 elif [ "$SITE" == "fram" ] ; then 
    echo "hardcoded settings for $SITE"
    export ESMF_DIR=/cluster/software/ESMF/7.1.0r-intel-2018b/
-   #export ESMF_DIR=/cluster/software/ESMF/6.3.0rp1-intel-2017a-HDF5-1.8.18/
+   export ESMF_MOD_DIR=${ESMF_DIR}mod/
+   export ESMF_LIB_DIR=${ESMF_DIR}lib/
+   
+elif [ "$SITE" == "surfsara" ] ; then 
+   echo "hardcoded settings for $SITE"
+   export ESMF_DIR=/sw/arch/RedHatEnterpriseServer7/EB_production/2019/software/ESMF/7.1.0r-intel-2018b/
    export ESMF_MOD_DIR=${ESMF_DIR}mod/
    export ESMF_LIB_DIR=${ESMF_DIR}lib/
    
 elif [[ "${unames:0:5}" == "Linux" ]] && [[ "$SITE" == "fram" ]] ; then
-   export ESMF_DIR=/cluster/software/ESMF/6.3.0rp1-intel-2017a-HDF5-1.8.18/
+   #export ESMF_DIR=/cluster/software/ESMF/6.3.0rp1-intel-2017a-HDF5-1.8.18/
```

Comment:
I would simply remove the old path.

Reply:
@Jipingnersc, Regarding ESMF version I am also using version 6.3.0rp1, but I also noticed that you have tested with a more recent version. I wonder if there is any difference/recommendation.